msi-ec.c

```diff
@@ -45,7 +48,8 @@ static int ec_read_seq(u8 addr, u8 *buf, int len)
 {
 	int result;
 	u8 i;
-	for (i = 0; i < len; i++) {
+	for (i = 0; i < len; i++)
```

Comment:
Open braces only for functions, not for loop or conditional structures: https://www.kernel.org/doc/html/v5.10/process/coding-style.html#placing-braces-and-spaces